### ISSUE TITLE
Icon animation

### ISSUE THREAD

**Issue by javico-dev11** [CodeSeven/toastr#702]
How could you add animation to the toast icon?

**Comment by jsvtrlvrslv** [comment]
I did it like this: an animated SVG icon, converted to base64, and placed it using CSS with a background URL.

**Comment by javico-dev11** [comment]
Thanks for your reply,
But with that you managed to make only the icon animate?

**Comment by jsvtrlvrslv** [comment]
Yes. Find one svg whit animation, convert to base64 and edit the CSS whit your base64 coded text. Easy like that. Look: 

https://github.com/CodeSeven/toastr/assets/86001662/0d8699bb-0bb7-4a90-9dcf-a426e0e7d34d
